### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by FranklinYinanDing**
Some minor changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,12 +17,13 @@ Sbt-restli is made up of 4 individual plugins.
 
 ### RestliSchemaPlugin
 
-The rest.li schema plugin compiles pegasus data-schemas (`.pdsc` files) into java data-templates.
+The rest.li schema plugin compiles pegasus data-schemas (`*.pdsc` files) into Java data-templates classes.
```
Comment (TylerHorth) [inline README.md]:
```suggestion
The rest.li schema plugin compiles pegasus data-schemas (`*.pdsc` files) into Java data-template classes.
```

File: README.md [hunk 2]
```diff
@@ -18,12 +17,13 @@ Sbt-restli is made up of 4 individual plugins.
...
 
-Apply the plugin to your project and place data-schemas in the `src/main/pegasus` directory.
+Apply the plugin to your project in `build.sbt` and place data-schemas in the `src/main/pegasus` directory.
 
 ```scala
-// build.sbt
+// TODO - add import for the plugin
```
Comment (TylerHorth) [inline README.md]: The `autoImport` object inside auto plugins is automatically glob imported by sbt, so no imports are needed from the end user. 

```suggestion
```

File: README.md [hunk 3]
```diff
@@ -38,12 +38,13 @@ lazy val api = (project in file("api"))
 
 The rest.li model plugin generates rest models (restspec & snapshot files) from your rest.li resource annotations, checks if they are compatible, and publishes them to your API project.
 
-Set your API project using `restliModelApi`, and set your compatibility mode using `restliModelCompat` to one of `OFF`, `IGNORE`, `BACKWARDS` (default), or `EQUIVALENT`. 
+Apply the plugin to your project in `build.sbt` and set your API project using `restliModelApi`, then set your compatibility mode using `restliModelCompat` to one of `OFF`, `IGNORE`, `BACKWARDS` (default), or `EQUIVALENT`. 
 
-Publishing your changes using `restliModelPublish` will copy rest models into your API project if they are compatible according to the compatibility mode selected.
+Publishing your changes using `restliModelPublish` task will copy rest models into your API project if they are compatible according to the compatibility mode selected.
```
Comment (TylerHorth) [inline README.md]:
```suggestion
Publishing your changes using the `restliModelPublish` task will copy rest models into your API project if they are compatible according to the compatibility mode selected.
```

File: README.md [hunk 4]
```diff
@@ -18,12 +17,13 @@ Sbt-restli is made up of 4 individual plugins.
...
-The rest.li schema plugin compiles pegasus data-schemas (`.pdsc` files) into java data-templates.
+The rest.li schema plugin compiles pegasus data-schemas (`*.pdsc` files) into Java data-templates classes.
 
-Apply the plugin to your project and place data-schemas in the `src/main/pegasus` directory.
+Apply the plugin to your project in `build.sbt` and place data-schemas in the `src/main/pegasus` directory.
 
 ```scala
-// build.sbt
+// TODO - add import for the plugin
+
```
Comment (TylerHorth) [inline README.md]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -38,12 +38,13 @@ lazy val api = (project in file("api"))
...
 
-Set your API project using `restliModelApi`, and set your compatibility mode using `restliModelCompat` to one of `OFF`, `IGNORE`, `BACKWARDS` (default), or `EQUIVALENT`. 
+Apply the plugin to your project in `build.sbt` and set your API project using `restliModelApi`, then set your compatibility mode using `restliModelCompat` to one of `OFF`, `IGNORE`, `BACKWARDS` (default), or `EQUIVALENT`. 
 
-Publishing your changes using `restliModelPublish` will copy rest models into your API project if they are compatible according to the compatibility mode selected.
+Publishing your changes using `restliModelPublish` task will copy rest models into your API project if they are compatible according to the compatibility mode selected.
 
 ```scala
-// build.sbt
+// TODO - add import for the plugin
```
Comment (TylerHorth) [inline README.md]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -38,12 +38,13 @@ lazy val api = (project in file("api"))
...
+
```
Comment (TylerHorth) [inline README.md]:
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -56,11 +57,13 @@ lazy val server = (project in file("server"))
...
 
-The rest.li client plugin generates java client bindings from rest models.
+The rest.li client plugin generates Java client bindings from rest models.
+
+Apply the plugin to your project in `build.sbt`. It is possible to apply the plugin to your API project directly, but it is best practice to create a new project in order to produce separate artifacts.
 
-It is possible to apply the plugin to your API project directly, but it is best practice to create a new project in order to produce separate artifacts.
 ```scala
-// build.sbt
+// TODO - add import for the plugin
```
Comment (TylerHorth) [inline README.md]:
```suggestion
```

File: README.md [hunk 8]
```diff
@@ -56,11 +57,13 @@ lazy val server = (project in file("server"))
...
+
```
Comment (TylerHorth) [inline README.md]:
```suggestion
```

**Review by TylerHorth — CHANGES_REQUESTED** [review]

**Comment by FranklinYinanDing** [comment]
Updated! Thanks!

**Pull request merged**